OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/PrefManager.java

```diff
@@ -283,6 +287,9 @@ public void setIsVideosCacheRestored(boolean restored) {
         public static final String AppNotificationPushHash = "AppNotificationPushHash";
         public static final String AppUpgradeNeedSyncWithParse = "AppUpgradeNeedSyncWithParse";
         public static final String AppSettingNeedSyncWithParse = "AppSettingNeedSyncWithParse";
+        public static final String UserPrefVideoModel = "UserPrefVideoModel";
+        public static final String LAST_COURSE_STRUCTURE_FETCH = "LastCourseStructureFetch";
```

Comment:
Another 2 changes that I am not sure how they got in here. @farhan 

This PR has been open to long.